### ISSUE TITLE
Final Report Peer Review - eer48

### ISSUE THREAD

**Issue by easonrecto** [tinali0923/Volatility-Prediction#14]
The goal of this report is to predict the volatility of stock returns using features such as Asset Turnover, PE, and Closing Price. The team started out their project by exploring their data set. They visualized relationships between features and outcomes and between features themselves. Using information gathered from data exploration, they cleaned the data set and used PCA to fill in missing entries. Next, they ran simple linear regressions for each feature as well as created a covariance matrix to get an even better understanding of the relationships amongst variables.  Finally, they decided to run three different types of Linear Regressions including: Lasso, Ridge, and Huber with quadratic regularizer. They made sure to choose the best lambda according to cross-validation and ultimately chose Lasso to be the best method.
This project was one of the most technically sound ones I have reviewed. This group made great decisions such as standardizing the data since the numbers were clustered in different regions and creating a covariance matrix to measure correlation between variables. I also thought they applied PCA very well in filling their missing entries. Another great aspect of this report were the visuals, which were both meaningful and aesthetically pleasing. I'm interested in the design choice to choose a 2:1 train test split. Because there is so much data (>100000) rows, I think the fit could be improved (without overfitting) with much less in the test set e.g. 85:15 which would still have 15,000 plus records in the test set which is over 10 times the size of some groups' full data sets! I would also like to know why Linear Regression was chosen over other classes of models such as Polynomial Regression or SVM. Also, it wouldn't be a bad idea to try out an unsupervised learning method to predict such as neural networks to look for different mixes or sequences of features to explain outcomes. Finally, I would have liked a little more out of the conclusion in terms of "so what?" Would you guys be willing to use this model in production to change how your company makes decisions?
Overall, very impressive report, great job!
